src/entity-manager/EntityManager.ts

```diff
@@ -177,6 +178,25 @@ export class EntityManager {
         return this.connection.query(query, parameters, this.queryRunner)
     }
 
+    /**
+     * Tagged template function that executes raw SQL query and returns raw database results.
+     * Template expressions are automatically transformed into database parameters.
+     * Note: Don't call this as a regular function, it is meant to be used with backticks to tag a template literal.
```

Comment:
I suppose we should mention Mongo here as well
```suggestion
     * Raw query execution is supported only by relational databases (MongoDB is not supported).
     * Note: Don't call this as a regular function, it is meant to be used with backticks to tag a template literal.
```